### PR TITLE
Qaart 703

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
Changed test for email to use a single email box. Test is now faking email change, by putting "+" characters before "@" symbol. Tests are working stable, 10 times in a row. There is a slight change of a test crashing into each other while executing in parallel, but it should be fixed with another ticket.

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java [hunk 1]
```diff
@@ -1,11 +1,14 @@
 package com.wikia.webdriver.testcases.specialpagestests;
 
+import org.apache.commons.lang3.StringUtils;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java]: MINOR Remove this unused import 'org.apache.commons.lang3.StringUtils'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MINOR 3 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialContactGeneralPage'.](https://github.com/Wikia/selenium-tests/blob/cdaf00f6f1de3257ac5293da28118735119327d8/src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java#L16) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/cdaf00f6f1de3257ac5293da28118735119327d8/src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java#L7) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/common/core/MailFunctions.java [hunk 2]
```diff
@@ -136,4 +136,19 @@ public static String getPasswordFromEmailContent(String mailContent) {
...
+   * @param emailAddress
+   * @return
+   */
+  public static String getEmail(String emailAddress) {
+
+    int specialsToAdd = 1 + StringUtils.countMatches(emailAddress, "+") % 3;
+    return emailAddress.replace("+", "").replace("@",
+        new String(new char[specialsToAdd]).replace("\0", "+") + "@");
+  }
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/core/MailFunctions.java]: remove this line

File: src/test/java/com/wikia/webdriver/common/core/MailFunctions.java [hunk 3]
```diff
@@ -136,4 +136,19 @@ public static String getPasswordFromEmailContent(String mailContent) {
...
+  /**
+   * Method generates a new "fake" email, by adding number of "+" characters before "@" symbol,
+   * this way, you can trick system that you changed email, but in fact,
+   * message will arrive to the same address
+   * @param emailAddress
+   * @return
+   */
+  public static String getEmail(String emailAddress) {
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/core/MailFunctions.java]: remove this line

File: src/test/java/com/wikia/webdriver/common/core/MailFunctions.java [hunk 4]
```diff
@@ -136,4 +136,19 @@ public static String getPasswordFromEmailContent(String mailContent) {
...
+    int specialsToAdd = 1 + StringUtils.countMatches(emailAddress, "+") % 3;
+    return emailAddress.replace("+", "").replace("@",
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/core/MailFunctions.java]: please provide line above return

**Comment by tomasznapieralski** [comment]
CR done

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**